tauntaun_live_editor/camp.py

```diff
@@ -39,6 +40,22 @@ def __init__(self, campaign):
         self.campaign: Campaign = campaign                
         self.group_route_request_handler = GameService.GroupRouteRequestHandler(campaign)
 
+    def add_jtac(self, coalition, countryName, location):
+        logging.debug(f"add_jtac {coalition} {countryName} {location}")

```

Comment:
Is it possible to add USA unit to other countries like Russia?
If yes then nothing to do, else check for countryName == USA then reject (with a logging).